serverless/tensorflow/matterport/mask_rcnn/nuclio/model_loader.py

```diff
@@ -1,49 +1,44 @@
-# Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Could you please keep the header and update it slightly (2020-2021)?

Reply:
done